backend/.gitignore

```diff
@@ -0,0 +1,134 @@
+node_modules
+.env
```

Comment:
.env is already mentioned on line 80

Reply:
and it seems node_modules is also already present on line 46

Reply:
fair observation, thanks!